packages/dd-trace/src/config.js

```diff
@@ -33,6 +34,11 @@ class Config {
     const reportHostname = coalesce(options.reportHostname, platform.env('DD_TRACE_REPORT_HOSTNAME'), false)
     const scope = coalesce(options.scope, platform.env('DD_TRACE_SCOPE'))
     const clientToken = coalesce(options.clientToken, platform.env('DD_CLIENT_TOKEN'))
+    const tags = {}
+
+    tagger.add(tags, platform.env('DD_TAGS'))
+    tagger.add(tags, platform.env('DD_TRACE_TAGS'))
```

Comment:
where does this env come from? existing?

Reply:
It was to be consistent with other tracers, but I just realized that the env var is actually `DD_TRACE_GLOBAL_TAGS`, will update to support it. I think I'll still leave DD_TRACE_TAGS just in case someone uses that one by accident since it's more inline with other env vars.